cmd/kgctl/showconf.go

```diff
@@ -121,6 +121,19 @@ func runShowConfNode(_ *cobra.Command, args []string) error {
 	if err != nil {
 		return fmt.Errorf("failed to list peers: %v", err)
 	}
+	// Optain the Granularity by looking at the annotation of the first node.
+	if opts.granularity == mesh.AutoGranularity {
+		if len(ns) == 0 {
+			return errors.New("failed to optain granularity: could not get any nodes")
+		}
+		opts.granularity = mesh.Granularity(ns[0].Granularity)
+		switch opts.granularity {
+		case mesh.LogicalGranularity:
+		case mesh.FullGranularity:
+		default:
+			return fmt.Errorf("mesh granularity %v unsupported", opts.granularity)
```

Comment:
maybe we can make this a more complete phrase?
```suggestion
			return fmt.Errorf("mesh granularity %q is not supported", opts.granularity)
```

and same above.

Also, I think it's nice to quote the values that produce the error to show that it is something funny that came from the user